CMakeLists.txt

```diff
@@ -74,24 +76,21 @@ message(STATUS "R_SCRIPT_COMMAND = ${R_SCRIPT_COMMAND}")
 # Install R package hera
 # ======================
 
-message(STATUS "Installing R 📦 hera to ${R_HOME}/library")
-if(EMSCRIPTEN)
-    execute_process(COMMAND ${R_COMMAND} CMD INSTALL
-        --preclean --no-staged-install --no-byte-compile --no-test-load
-        --build ../hera --library=${R_HOME}/library
-        RESULT_VARIABLE hera_result
-    )
-
+if(CMAKE_CROSSCOMPILING)
```

Comment:
I would not rely on this variable, since it can be set to true even if CMAKE_SYSTEM_NAME and CMAKE_HOST_SYSTEM_NAME are the same:
> In the event that [CMAKE_SYSTEM_NAME](https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_NAME.html#variable:CMAKE_SYSTEM_NAME) is manually set to the same value as [CMAKE_HOST_SYSTEM_NAME](https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_SYSTEM_NAME.html#variable:CMAKE_HOST_SYSTEM_NAME), then CMAKE_CROSSCOMPILING will still be set to true.

Comparing these variables looks more robust